### PR TITLE
Corrections

### PR DESCRIPTION
**Pull request by zaneb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Let us familiarize ourselves with the Agent-Based Installer workflow.   Unlike t
...
 
-Before we begin lets ensure we are using RHEL8 or RHEL9 on x86 architecture as our location to build the binary and have installed the packages <code>go</code>, <code>make</code> and <code>zip</code>.  Further, we will want to ensure that <code>nmstatectl</code> is installed on the system.  We also need about 5G of available disk space for the source code, compiled binary and generated images. Now lets go grab the OpenShift installer source code from Github and checkout the specific tag agent-based-installer-4.11-developer-preview:
+Before we begin lets ensure we are using RHEL8 or RHEL9 on x86 architecture as our location to build the binary and have installed the packages <code>go</code>, <code>make</code> and <code>zip</code>.  Further, we will want to ensure that <code>nmstatectl</code> 2.0 is installed on the system.  We also need about 5G of available disk space for the source code, compiled binary and generated images. Now lets go grab the OpenShift installer source code from Github and checkout the specific tag agent-based-installer-4.11-developer-preview:
```
Comment (zaneb) [inline README.md]: @bfournie is this correct?

Comment (schmaustech) [reply README.md]: On the system I did testing installing nmstate-1.2.1-3.el8_6.x86_64 provided the nmstatectl I used.

Comment (zaneb) [reply README.md]: OK, yeah I expect anything earlier than 2.1 would work then. Maybe we should just say 1.x?

Comment (zaneb) [reply README.md]: I changed it to 1.x.

**Pull request merged**